library/std/src/sys/windows/fs.rs

```diff
@@ -1393,6 +1393,8 @@ fn symlink_junction_inner(original: &Path, junction: &Path) -> io::Result<()> {
         let mut data = Align8([MaybeUninit::<u8>::uninit(); c::MAXIMUM_REPARSE_DATA_BUFFER_SIZE]);
```

Comment:
Might [`MU::zeroed`](https://doc.rust-lang.org/std/mem/union.MaybeUninit.html#method.zeroed) be clearer here?
```suggestion
        let mut data = Align8([MaybeUninit::<u8>::zeroed(); c::MAXIMUM_REPARSE_DATA_BUFFER_SIZE]);
```

Reply:
Hm, that would zero the entire buffer tho. Which I assume we're trying to avoid because it's large. Otherwise `[0_u8, c::MAXIMUM_REPARSE_DATA_BUFFER_SIZE]` would probably be ok because it's a plain old data struct.

Reply:
It probably doesn't matter too much if it's only used in tests but good point.

Reply:
It's apparently 16KB for the buffer, so we'd be touching 4 pages on x86. That doesn't thrash all of a standard 32KB cache but it's still a lot when compared to simply zeroing a few shorts and longs.

otoh, "just tests".